dfd_tests/urls.py

```diff
@@ -1,6 +1,9 @@
 from django.conf.urls import include, url
 from django.contrib import admin
+import django
+
+admin_urls = (admin.site.urls if (django.VERSION[0] == 2) else include(admin.site.urls))
 
 urlpatterns = [
-    url(r'^admin/', include(admin.site.urls)),
+	url(r'^admin/', admin_urls),
```

Comment:
Why the indentation change?

Reply:
No idea. Must've been the editor, not me. 😇